ts/IterableInbox.tsx

```diff
@@ -1,30 +1,383 @@
 'use strict'
 
 import React, { useState } from 'react'
-import { Text, View, StyleSheet } from 'react-native'
+import { Text, SafeAreaView, StyleSheet } from 'react-native'
+
 import IterableInboxMessageList from './IterableInboxMessageList'
 import IterableInboxEmptyState from './IterableInboxEmptyState'
 import IterableInboxMessageDisplay from './IterableInboxMessageDisplay'
-import sampleMessages from './sampleMessageData.js'
 import Message from './messageType'
 
+let sampleMessages = [
```

Comment:
how do you feel about importing this mock data from another file. putting it here makes this file needlessly larger than it needs to be.

Reply:
Haha, I was gonna say, I'm not sure why Evan made this change, but I'll note that we'll be getting rid of this soon anyways, so take that as you will.